Merge #276 broke graphql-helix-defer-stream example

**Describe the bug**

It appears merge #276 has broken examples due to context being passed to `getEnveloped()`.

<details>
<summary>Error:</summary>

```bash
$ yarn start
yarn run v1.22.15
warning ../../../../../package.json: No license field
$ ts-node index.ts

/home/levi/Desktop/Projects/graphql-playground/graph-ql-evelop/graphql-helix-defer-stream/node_modules/ts-node/src/index.ts:587
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
index.ts:151:79 - error TS2554: Expected 0 arguments, but got 1.

151     const { parse, validate, contextFactory, execute, schema } = getEnveloped({ req });
                                                                                  ~~~~~~~

    at createTSError (/home/levi/Desktop/Projects/graphql-playground/graph-ql-evelop/graphql-helix-defer-stream/node_modules/ts-node/src/index.ts:587:12)
    at reportTSError (/home/levi/Desktop/Projects/graphql-playground/graph-ql-evelop/graphql-helix-defer-stream/node_modules/ts-node/src/index.ts:591:19)
    at getOutput (/home/levi/Desktop/Projects/graphql-playground/graph-ql-evelop/graphql-helix-defer-stream/node_modules/ts-node/src/index.ts:921:36)
    at Object.compile (/home/levi/Desktop/Projects/graphql-playground/graph-ql-evelop/graphql-helix-defer-stream/node_modules/ts-node/src/index.ts:1189:32)
    at Module.m._compile (/home/levi/Desktop/Projects/graphql-playground/graph-ql-evelop/graphql-helix-defer-stream/node_modules/ts-node/src/index.ts:1295:42)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.require.extensions.<computed> [as .ts] (/home/levi/Desktop/Projects/graphql-playground/graph-ql-evelop/graphql-helix-defer-stream/node_modules/ts-node/src/index.ts:1298:12)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
error Command failed with exit code 1.
```
</details>

<details>
<summary>package.json</summary>

```JSON
{
  "name": "@envelop-examples/graphql-helix-defer-stream",
  "private": true,
  "version": "1.0.0",
  "main": "index.js",
  "author": "Dotan Simha",
  "license": "MIT",
  "dependencies": {
    "fastify": "3.14.0",
    "@envelop/core": "0.2.1",
    "graphql-helix": "1.6.1",
    "@graphql-tools/schema": "8.0.1",
    "graphql": "experimental-stream-defer"
  },
  "devDependencies": {
    "@types/node": "15.6.1",
    "ts-node": "10.0.0",
    "typescript": "4.3.5"
  },
  "scripts": {
    "start": "ts-node index.ts"
  }
}


```
</details>

**To Reproduce**
Steps to reproduce the behavior:

1. Clone repo
2. Follow directions in [graphql-helix-defer-stream example](https://github.com/dotansimha/envelop/tree/main/examples/graphql-helix-defer-stream) 

**Expected behavior**

Expected not to encounter TS error.

**Environment:**

- OS: `Pop!_OS 20.10`
- `@envelop-core`: `0.2.1`
- NodeJS: `v14.16.0`

--- a/examples/graphql-helix-defer-stream/package.json
+++ b/examples/graphql-helix-defer-stream/package.json
@@ -6,16 +6,16 @@
   "author": "Dotan Simha",
   "license": "MIT",
   "dependencies": {
-    "fastify": "3.14.0",
-    "@envelop/core": "0.2.1",
+    "fastify": "3.22.0",
+    "@envelop/core": "1.3.0",
     "graphql-helix": "1.8.3",
-    "@graphql-tools/schema": "8.0.1",
+    "@graphql-tools/schema": "8.2.0",
     "graphql": "experimental-stream-defer"
   },
   "devDependencies": {
-    "@types/node": "15.6.1",
-    "ts-node": "10.0.0",
-    "typescript": "4.3.5"
+    "@types/node": "16.10.9",
+    "ts-node": "10.3.0",
+    "typescript": "4.4.4"
   },
   "scripts": {
     "start": "ts-node index.ts"

